src/assets/styles/vuetifyTheme.js

```diff
@@ -103,33 +103,33 @@ export default {
       },
       dark: {
       // new colors
-        backgroundWallet: '#F2F4FA',
-        backgroundOverlay: '#F2FAFA',
-        backgroundGrey: '#F8F9FB',
+        backgroundWallet: '#000000',
+        backgroundOverlay: '#000000',
+        backgroundGrey: '#000000',
         whiteAlways: '#FFFFFF',
-        whiteBackground: '#FFFFFF',
-        whitePopup: '#FFFFFF',
-        textDark: '#192133',
-        textMedium: '#5A678A',
-        textLight: '#939FB9',
-        greyPrimary:'#5A678A',
-        greyMedium: '#D7DAE3',
-        greyLight: '#F2F3F6',
-        greenPrimary: '#05C0A5',
-        greenMedium: '#C3F0E9',
-        greenLight: '#EBFAF8',
-        bluePrimary: '#4B83E8',
-        blueMedium: '#D4E1F9',
-        blueLight: '#EEF3FD',
-        redPrimary: '#FF445B',
-        redMedium: '#FFD2D8',
-        redLight: '#FFF0F2',
-        orangePrimary: '#F5A623',
-        orangeMedium: '#FBDBA7',
-        orangeLight: '#FEF4E5',
-        disabledPrimary: '#CECECE',
-        disabledMedium: '#E0E0E0',
-        disabledLight: '#F6F6F6',
+        whiteBackground: '#192439',
+        whitePopup: '#202b43',
+        textDark: '#ccd4e6',
+        textMedium: '#626e86',
+        textLight: '#3a4562',
+        greyPrimary:'#5a678a',
+        greyMedium: '#323d57',
+        greyLight: '#26324a',
+        greenPrimary: '#06d9b9',
+        greenMedium: '#107e7a',
```

Comment:
greenMedium is  #06D9B9

Reply:
Fixed

Reply:
I think you changed _greenPrimary_ instead if **greenMedium**

Reply:
![Screenshot from 2021-12-21 16-11-36](https://user-images.githubusercontent.com/9893774/147013622-e6cf6824-bd98-4d50-8a3c-b7d053412f8c.png)
![Screenshot from 2021-12-21 16-11-41](https://user-images.githubusercontent.com/9893774/147013623-ed0000f4-78a3-4d73-8c91-e1aab28b1c44.png)

Green Primary and Medium has same values as you see on the picture (figma). I capture the real greenMedium value from chrome debug pannel, so it should be fine.